WikiDrips/WikiTableViewController.swift

```diff
@@ -179,16 +196,16 @@ extension WikiTableViewController: UITableViewDataSourcePrefetching {
         }
     }
     
-// No need for cancelPrefetchingForRowsAt since changing search cancels pending operations and prefetchRowsAt populates a cache
+    // No need for cancelPrefetchingForRowsAt since changing search cancels pending operations and prefetchRowsAt populates a cache
```

Comment:
A few things.. first, I think fetching only the first 10 records is too low. I think we the default page size should fill at least 1 screen full.. so I don't know if you can increase the page size, but if you can I would change it to 25 or 50. From a 'device power' perspective its better to fetch more data and make less requests.
Second, regarding pagination, I'd be curious to see how the scrolling feels with your approach (i.e. using the prefetch of the last record to trigger fetching the next page). On LSM Eldar didn't use prefetch but instead used the cellForRowAtIndex method. However, he didn't wait till we got to the last row, instead he triggers the next page fetch when we get a certain fraction down the first page.
I'm ok if you want to implement the cancelPrefetching in that PR instead.